package.json

```diff
@@ -187,7 +187,7 @@
     "build-devdocs:components-usage-stats:_env": "node server/devdocs/bin/generate-components-usage-stats.js \"client/**/*.js\" \"client/**/*.jsx\" \"!**/docs/**\" \"!**/test/**\" \"!**/docs-example/**\"",
     "build-devdocs:index": "npm run -s env -- npm run -s build-devdocs:index:_env",
     "build-devdocs:index:_env": "node server/devdocs/bin/generate-devdocs-index \"**/*.md\" \".github/**.md\" \"!build/**\" \"!node_modules/**\" \"!public/**\"",
-    "build-docker": "docker build -t wp-calypso .",
+    "build-docker": "docker build --build-arg commit_sha=`git rev-parse HEAD` -t wp-calypso .",
```

Comment:
does this work on windows?

Reply:
I didn't realize we were running the `docker build` command from a windows, non bash prompt. I suppose if that's the case, this will need to be changed.

Reply:
I can confirm that this doesn't work in the Windows command prompt. It doesn't work in the "Git Bash" either. I assume it *will* work in the Linux Subsystem though (I don't have it).

After talking with Kevin, I'm OK if this doesn't get fixed right now, as I think I'm the only Automattician using a plain Windows prompt for developing Calypso, and Docker isn't used much in development anyway.

The fix should be reasonable simple. Either change this line with a custom JS script that launches `git` and `docker` using `child_process`, or write the SHA1 into a file and then read it from inside Docker.